embedx/config.schema.json

```diff
@@ -2620,6 +2617,12 @@
           "title": "Enable Ory Sessions caching",
           "description": "If enabled allows Ory Sessions to be cached. Only effective in the Ory Network.",
           "default": false
+        },
+        "new_flow_transitions": {
```

Comment:
Should we call this `use_continue_with_transitions` or similar? `new_flow_transitions` is a bit ambiguous.

Reply:
Done.